Cargo.lock

```diff
@@ -19,6 +19,8 @@ version = "0.4.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
  "rustc-serialize 0.3.24 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde 1.0.11 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde_derive 1.0.11 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
The Cargo.lock changes don't belong in this PR, I think.

Reply:
I think they do, as the addition of the optional dependency on the feature means cargo needs to know about those dependencies, where it didn't before. I'm a bit unsure on this, so let me know if I'm wrong.

Reply:
👍